dreamjob_frontend/src/routes/jobDetails.js

```diff
@@ -23,9 +23,15 @@ class jobDetails extends Component {
     })
   }
 
+  renewJobs(){
```

Comment:
can't we just call updateJobs directly instead of calling one with in the other?